src/App.vue

```diff
@@ -48,6 +52,9 @@
 
 		<NcSettingsSection v-if="hasTerms"
 			:name="t('terms_of_service', 'Existing terms of service')">
+			<p class="terms-descr">
+				{{ t('terms_of_service', 'Only resend the notification if there are legal changes. For minor updates like fixing typos, avoid resending, as it will require all users to reaccept the Terms of Service.') }}
```

Comment:
```suggestion
				{{ t('terms_of_service', 'We recommend to reset signatures if legal changes were applied. For minor changes like fixing typos or correcting links, it could be left out, as it would otherwise require all users to accept the Terms of Service again.') }}
```